packages/ckeditor5-ui/src/textarea/textareaview.ts

```diff
@@ -193,3 +195,13 @@ function getTextareaElementClone( element: HTMLTextAreaElement, value: string ):
 
 	return clone;
 }
+
+/**
+ * Fired when the textarea view is resized.
```

Comment:
I'm wondering if `resized` is causing any confusion. In the context of textarea, `resize` is typically used when a user has changed the size.